test/integration/actions/action-create-event.spec.ts

```diff
@@ -72,6 +74,42 @@ describe('action-create-event', () => {
 			},
 		};
 
+		// Before enqueing up a create event we need to create a relationship to the random-slug type created above
+		const relationship: RelationshipContractDefinition = {
```

Comment:
Could we instead change the types used in the test so we can use a real relationship we expect to exist?

Reply:
The comment here is misleading, in this case, the test needs the `relationship-card-is-attached-to-card` relationship to be present. That relationship is currently not part of the `SDK-link-constraints`. This relationship is also needed by other tests in the same module, so I've refactored the code to move the creation of this relationship to the `beforeAll` method.

About the main question, I'll review this after refactoring the relationship creation ( moving relationships to their plugin )